dothething/dothething/ViewModels/ClipsViewModel.swift

```diff
@@ -18,23 +18,25 @@ struct Clip: Hashable {
     let thumbnail: UIImage
     let isHighlighted: Bool
     let metadata: ClipMetadata
+    let showCode: Bool
     
-    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata) {
+    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata, showCode: Bool) {
         self.url = url
         self.thumbnail = thumbnail
         self.isHighlighted = isHighlighted
         self.metadata = metadata
+        self.showCode = showCode
```

Comment:
The addition of the "showCode" property to the Clip struct seems reasonable, as it allows for the display of code snippets related to each clip. However, it's worth considering whether this property should be part of the Clip struct or if it would be better suited as a separate property in the ClipsViewModel. This separation of concerns could improve the overall design and maintainability of the code.